students/jemielity_kamil/lesson_11_argparse_and_math/filling_the_gaps.py

```diff
@@ -0,0 +1,34 @@
+import os
+import re
+import argparse
+
+parser = argparse.ArgumentParser(description='Filling the gaps')
+parser.add_argument('-p', action='store',
+                    dest="prefix", required=True,
+                    help='prefix')
+args = parser.parse_args()
+
+
+prefix = args.prefix
+SUFFIX = '.txt'
```

Comment:
Świetnie że nazwałeś stałe wielkimi literami, jednak powinny one wylądować na górze pliku pod importami: https://www.python.org/dev/peps/pep-0008/#imports
Tak mówi PEP8 i to jest miejsce, w którym większość developerów będzie ich szukać.